Code/GraphMol/ChemReactions/ReactionParser.h

```diff
@@ -36,9 +36,11 @@
 #ifndef RD_REACTIONPARSER_H_21Aug2006
 #define RD_REACTIONPARSER_H_21Aug2006
 
+#include <GraphMol/ROMol.h>
```

Comment:
It's not clear to me why this file would need `ROMol.h` to actually be included. Can't you just add a forward declaration of the `ROMol` class before the forward declaration of `ChemicalReaction` (around line 50)?

In order to keep compile times down, it's generally good to avoid including headers in other headers unless they are actually required.

Reply:
Including `ROMol.h` is be consistent with the [isocpp guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf11-header-files-should-be-self-contained) and [google c++ style guide](https://google.github.io/styleguide/cppguide.html#Forward_Declarations). It would indeed speed up a little bit the build time, but it can make the code more brittle. Happy to make the change if you think it's best.

Reply:
Fair enough, I incorporated the change.